Configuration/PyReleaseValidation/python/relval_data_highstats.py

```diff
@@ -12,50 +12,56 @@
 offset_events = 0.0001 # less than 10 event setups (50k,150k,250k,500k)
 
 ## 2024
-base_wf_number_2024 = 2024.0
+base_wf = 2024.0
 for e_n,era in enumerate(eras_2024):
     for p_n,pd in enumerate(pds_2024):
         for e_key,evs in event_steps_dict.items():
             if "10k" == e_key: # already defined in relval_standard
                 continue   
-            wf_number = base_wf_number_2024
+            wf_number = base_wf
             wf_number = wf_number + offset_era * e_n
             wf_number = wf_number + offset_pd * p_n
-            wf_number = wf_number + offset_events * evs 
+            wf_number = wf_number + 0.0001 * 0.01 
```

Comment:
Just curious: why did you move to the hardcoded numbers rather than sticking on the parameterized ones? Why do you do so for all eras but the `eras_2022_2` one?

Reply:
It was a stupid copy paste. Thanks for spotting it. Just reverted.